plutus-tx-plugin/test/Budget/9.2/lte0.budget.golden

```diff
@@ -0,0 +1,2 @@
+({cpu: 2206118053
+| mem: 6212402})
```

Comment:
As expected `<= 0` is a teeny tiny bit faster than `>= 0` (because there's a builtin for the former, but not the latter, which is defined via the former + `not`). The stats for `gte0` are

```
({cpu: 2210128053
| mem: 6212402})
```

I'm not sure I understand how `mem` is exactly the same though.

Reply:
I'm actually surprised that the CPU is different, since `x<=0` is `\x -> ifThenElse (x<=0) True False` and `x>=0` is `\x -> ifThenElse (x<0) False True` which should be the same.

Reply:
Oh, `<=` and `<` have different CPU cost parameters, that's why!